sdk-api-src/content/synchapi/nf-synchapi-createmutexa.md

```diff
@@ -147,8 +145,8 @@ Multiple processes can have handles of the same mutex object, enabling use of th
 <b>CreateMutex</b> enabled inheritance. This mechanism works for both named and unnamed mutexes.</li>
 <li>A process can specify the handle to a mutex object in a call to the <a href="https://docs.microsoft.com/windows/desktop/api/handleapi/nf-handleapi-duplicatehandle">DuplicateHandle</a> function to create a duplicate handle that can be used by another process. This mechanism works for both named and unnamed mutexes.</li>
 <li>A process can specify a named mutex in a call to the 
-[OpenMutex](/windows/win32/api/synchapi/nf-synchapi-openmutexw)a> or 
-<b>CreateMutex</b> function to retrieve a handle to the mutex object.</li>
+<a href="https://docs.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-openmutexw">OpenMutex</a> or 
```

Comment:
[Links from one article to another](https://docs.microsoft.com/contribute/how-to-write-links#links-from-one-article-to-another) recommends not including locales such as "en-us" in URLs for Microsoft properties.